lib/modules/manager/gradle/extract.spec.ts

```diff
@@ -232,7 +232,7 @@ describe('modules/manager/gradle/extract', () => {
 
     mockFs({
       'build.gradle': buildFile,
-      'gradle.properties': 'repositoryBaseURL: https://dummy.org/whatever',
+      'gradle.properties': 'repositoryBaseURL: https\\://dummy.org/whatever',
```

Comment:
Does this need to be a double `\`?

Reply:
- In a real `gradle.properties` you could have a single `\`
- In this test the double `\` is intended, as a single `\` would escape the colon and it is really about the `\` that occurs as character